src/channels/channelsDialogue.component.test.tsx

```diff
@@ -183,4 +183,18 @@ describe('Channels Dialogue', () => {
     ]);
     expect(props.onClose).toHaveBeenCalled();
   });
+
+  it('selects a channel in the metadata panel & channel tree when search box is used', async () => {
+    createView();
+
+    const search = screen.getByLabelText('Search data channels');
+
+    await user.type(search, 'shot{arrowdown}{enter}');
+
+    // shot number should be visible in tree view
+    screen.getByRole('checkbox', { name: 'Shot Number' });
```

Comment:
Should this be wrapped in an `expect` assertion?
```js
expect(screen.getByRole('checkbox', { name: 'Shot Number' })).toBeInTheDocument();
```

Reply:
Technically it doesn't have to be, as `getByRole` would error if it wasn't in the document, but I can wrap in `expect` to make it more explicit

Reply:
Yeah, I think wrapping it in an expect makes the intention clearer, especially for devs who might not be aware that `getBy*` queries throw when no element is found.